src/sentry/static/sentry/app/components/avatarCropper.jsx

```diff
@@ -368,7 +368,7 @@ const AvatarCropper = React.createClass({
   render() {
     let src = this.getImgSrc();
     let style = {
-      position: 'absolute',
```

Comment:
I did some quick tests, and It looks like this is set to `opacity: 0;` to keep it in the tab order ([see here for reference](https://stackoverflow.com/questions/272360/does-opacity0-have-exactly-the-same-effect-as-visibilityhidden)). While the focus state isn't super clear, I think it is a win to keep this accessible to keyboard only users. 

I do think it would be clearer and better, however, if the css we have applied to the `id` attribute in `shared-components.less` were just applied here. We wouldn't be able to use Emotion but at least it would be close lol.

Reply:
Why can't you use emotion?

Reply:
👍